docs/database/ftl.md

```diff
@@ -51,6 +51,8 @@ Label | Type | Allowed to by empty | Content
 `forward` | text | Yes | Forward destination used for this query (only set if `status == 2`)
 `additional_info` | blob | Yes | Data-dependent content, see below
 
+The `queries` `VIEW` is dynamically generated from the data actually stored in the tables `queries_storage` and the linking tables `domain_by_id`, `client_by_id`, `forward_by_id`, and `addinfo_by_id` (see below). The table `queries_storage` will contains integer IDs pointing to the respective entries of the linking tables to save space and make searching the database faster. If you haven't upgraded for some time, the table may still contain strings instead of integer IDs.
+
 #### Data-dependent `additional_info` field
```

Comment:
This can go to `addinfo_by_id` now.

Reply:
I disagree for the same reasons as outlined [here](https://github.com/pi-hole/docs/pull/645#issuecomment-1021757398). Users are supposed to interact with the database on a sufficiently high level to not cause unnecessary work for them and open the potential for potential (and unnecessary) harm when they interact with the database on the lower levels (while this isn't necessary).

Reply:
Ok, convinced.